ops/install-spark.sh

```diff
@@ -8,7 +8,7 @@ readonly fortis_central_directory="$3"
 readonly gh_fortis_spark_repo_name="project-fortis-spark"
 readonly latest_version=$(curl "https://api.github.com/repos/catalystcode/${gh_fortis_spark_repo_name}/releases/latest" | jq -r '.tag_name')
 readonly fortis_jar="fortis-${latest_version}.jar"
-readonly SparkCommand="wget \"https://fortiscentral.blob.core.windows.net/jars/${fortis_jar}\" && spark-submit --driver-memory 4g --supervise --master spark://spark-master:7077 --verbose --class com.microsoft.partnercatalyst.fortis.spark.ProjectFortis ${fortis_jar}"
+readonly SparkCommand="spark-submit --deploy-mode cluster --driver-memory 4g --supervise --master spark://spark-master:7077 --verbose --class com.microsoft.partnercatalyst.fortis.spark.ProjectFortis \"https://fortiscentral.blob.core.windows.net/jars/${fortis_jar}\""
```

Comment:
Ah yeah it'll pull it down. I'm not sure if 'cluster' mode is required, but if it's an `http(s)` URI, it'll fetch it. It's in the Spark programming guide somewhere :)